docs/vdc/networking-virtual-datacenter.md

```diff
@@ -10,360 +10,375 @@ ms.date: 09/24/2018
 
-## Virtual Data Center overview
+## Virtual datacenter overview
 
 ### Topology
-_Hub and spokes_ is a model for extending a Virtual Data Center within a single Azure region.
+**Hub and spokes** is a model for extending a virtual datacenter within a single Azure region.
 
 [![1]][1]
 
-The hub is the central zone that controls and inspects ingress and/or egress traffic between different zones: Internet, on-premises, and the spokes. The hub and spoke topology gives the IT department an effective way to enforce security policies in a central location, while reducing the potential for misconfiguration and exposure.
+The hub is the central zone that controls and inspects ingress or egress traffic between different zones: internet, on-premises, and the spokes. The hub and spoke topology gives the IT department an effective way to enforce security policies in a central location. It also reduces the potential for misconfiguration and exposure.
 
-The hub contains the common service components consumed by the spokes. Here are a few typical examples of common central services:
+The hub contains the common service components consumed by the spokes. The following examples are common central services:
 
--   The Windows Active Directory infrastructure (with the related ADFS service) required for user authentication of third parties accessing from untrusted networks before getting access to the workloads in the spoke
--   A DNS service to resolve naming for the workload in the spokes, to access resources on-premises and on the Internet if [Azure DNS][DNS] isn't used
--   A PKI infrastructure, to implement single sign-on on workloads
--   Flow control (TCP/UDP) between the spokes and Internet
--   Flow control between the spoke and on-premises
--   If desired, flow control between one spoke and another
+-   The Windows Active Directory infrastructure, required for user authentication of third parties that access from untrusted networks before they get access to the workloads in the spoke. It includes the related Active Directory Federation Services (AD FS).
+-   A DNS service to resolve naming for the workload in the spokes, to access resources on-premises and on the internet if [Azure DNS][DNS] isn't used.
+-   A public key infrastructure (PKI), to implement single sign-on on workloads.
+-   Flow control (TCP, UDP) between the spokes and internet.
```

Comment:
Should be "Flow control of TCP and UDP traffic between the spoke network zones and the internet."